{
  "channeldata": {
    "activate.d": false,
    "binary_prefix": false,
    "deactivate.d": false,
    "description": null,
    "dev_url": null,
    "doc_source_url": null,
    "doc_url": null,
    "home": "https://github.com/landscapeio/pylint-django",
    "icon_hash": null,
    "icon_url": null,
    "identifiers": null,
    "keywords": null,
    "license": "GPL 2",
    "post_link": false,
    "pre_link": false,
    "pre_unlink": false,
    "recipe_origin": null,
    "run_exports": {},
    "source_git_url": null,
    "source_url": "https://pypi.io/packages/source/p/pylint-django/pylint-django-0.7.2.tar.gz",
    "subdirs": [
      "win-64"
    ],
    "summary": "pylint-django is a Pylint plugin to aid Pylint in recognising and understandingerrors caused when using the Django framework",
    "tags": null,
    "text_prefix": false,
    "timestamp": 0,
    "version": "0.7.2"
  },
  "channeldata_version": 1,
  "feedstock": null,
  "labels": [
    "main",
    "cf202003"
  ],
  "package": "pylint-django-0.7.2-py34_0.tar.bz2",
  "repodata": {
    "build": "py34_0",
    "build_number": 0,
    "depends": [
      "pylint >=1.0",
      "pylint-plugin-utils >=0.2.1",
      "python 3.4*"
    ],
    "license": "GPL 2",
    "md5": "feb18f49e60a82422d3fffb910db21b0",
    "name": "pylint-django",
    "sha256": "091e593c5a50a24a65f901fd90a9bb7ac22bca607d8a740d0f8252c6a2654ab4",
    "size": 22474,
    "subdir": "win-64",
    "version": "0.7.2"
  },
  "repodata_version": 1,
  "subdir": "win-64",
  "url": "https://conda.anaconda.org/conda-forge/win-64/pylint-django-0.7.2-py34_0.tar.bz2"
}